src/App.tsx

```diff
@@ -1,60 +1,135 @@
+import { useState } from 'react';
+
 import './App.scss';
+import { TodoList } from './components/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [userIdAdd, setUserId] = useState(0);
+  const [titleAdd, setTitle] = useState('');
+  const [errorTitle, setErrorTitle] = useState(false);
+  const [errorUser, setErrorUser] = useState(false);
+
+  function Errors() {
```

Comment:
function name shouldn't be capitalize, and also naming should be more explicit, and explains what function do